diesel/src/associations/mod.rs

```diff
@@ -122,6 +122,19 @@ use query_source::Table;
 
 pub use self::belongs_to::{BelongsTo, GroupedBy};
 
+/// Represents a struct which can be identified on a single table in the
+/// database. This must be implemented to use associations, and some features of
+/// updating.
+///
+/// ### Deriving
+///
+/// This trait can be automatically derived using `diesel_codegen` by adding
+/// `#[derive(Identifiable)]` to your struct. The primary key will be assumed to
+/// be a field and column called `id`. By default the table will be
+/// assumed to be the plural form of the struct name (using *very* dumb
+/// pluralization). If your table name differs from that convention, or requires
```

Comment:
If the convention is so dumb you can probably describe it half a sentence, e.g.

> using *very* dumb pluralization -- it just adds an 's' at the end

I'm also wondering if this is something we want to guarantee any kind of stability of (which we should mention regardless of yes or no).